module/applications/item/item-sheet.mjs

```diff
@@ -269,77 +268,6 @@ export default class ItemSheet5e extends ItemSheet {
 
   /* -------------------------------------------- */
 
-  /**
-   * Get the valid item consumption targets which exist on the actor
-   * @returns {Object<string>}   An object of potential consumption targets
-   * @private
-   */
-  _getItemConsumptionTargets() {
-    const consume = this.item.system.consume || {};
-    if ( !consume.type ) return [];
-    const actor = this.item.actor;
-    if ( !actor && (consume.type !== "hitDice") ) return {};
-
-    // Ammunition
```

Comment:
How are we planning on handling ammo?

Reply:
I believe ammunition should be handled specially by the `AttackActivity` on weapons, because it is consumed when attacks are made and now when the activation is performed.

Reply:
That makes a lot of sense, yeah